rfc-177-edition-all-content.md

```diff
@@ -0,0 +1,148 @@
+    FeaturedImageData {}
+    WorldLocation {}
+    Organisation {}
+    Role {}
+    WorldNewsStory["WorldNewsStory (Edition)"] {}
+    CorporateInformationPage["CorporateInformationPage (Edition)"] {}
+
+    WorldwideOrganisation ||--|{ WorldwideOrganisationWorldLocation : "has many"
+    WorldwideOrganisation ||--|{ SocialMediaAccount : "has many"
+    WorldwideOrganisation ||--|{ Sponsorship : "has many"
+    WorldwideOrganisation ||--|{ WorldwideOffice : "has many"
+    WorldwideOrganisation ||--|{ WorldwideOrganisationRole : "has many"
+    WorldwideOrganisation ||--|{ EditionWorldwideOrganisation : "has many"
+    WorldwideOrganisation ||--|{ FeaturedImageData : "has many"
+
+    WorldwideOrganisationWorldLocation ||--|| WorldLocation : "has one"
+    Sponsorship ||--|| Organisation : "has one"
+    WorldwideOrganisationRole ||--|| Role : "has one"
+    EditionWorldwideOrganisation ||--|| WorldNewsStory : "has one"
+    EditionWorldwideOrganisation ||--|{ CorporateInformationPage : "has many"
+```
+
+The diagram below shows the relationship between models in Whitehall after Worldwide Organisations were made editionable:
+
+```mermaid
+---
+title: "Architecture of worldwide organisation model's relationship to other models after being made editionable in Whitehall"
+---
+
+erDiagram
+    Attachment {}
+    Document {}
+    EditionableWorldwideOrganisation["EditionableWorldwideOrganisation (Edition)"] {}
+    EditionEditionableWorldwideOrganisation {}
+    EditionOrganisation {}
+    EditionRole {}
+    EditionWorldLocation {}
+    FeaturedImageData {}
+    Organisation {}
+    Role {}
+    SocialMediaAccount {}
+    WorldLocation {}
+    WorldNewsStory["WorldNewsStory (Edition)"] {}
+    WorldwideOffice {}
+    WorldwideOrganisationPage {}
+
+    Document ||--|{ EditionableWorldwideOrganisation : "has many"
+
+    EditionableWorldwideOrganisation ||--|{ EditionWorldLocation : "has many"
+    EditionableWorldwideOrganisation ||--|{ SocialMediaAccount : "has many"
+    EditionableWorldwideOrganisation ||--|{ EditionOrganisation : "has many"
+    EditionableWorldwideOrganisation ||--|{ WorldwideOffice : "has many"
+    EditionableWorldwideOrganisation ||--|{ EditionRole : "has many"
+    EditionableWorldwideOrganisation ||--|{ EditionEditionableWorldwideOrganisation : "has many"
+    EditionableWorldwideOrganisation ||--|{ WorldwideOrganisationPage : "has many"
+    EditionableWorldwideOrganisation ||--|{ FeaturedImageData : "has many"
+    EditionableWorldwideOrganisation ||--|{ Attachment : "has many"
+
+    EditionWorldLocation ||--|| WorldLocation : "has one"
+    EditionOrganisation ||--|| Organisation : "has one"
+    EditionRole ||--|| Role : "has one"
+    EditionEditionableWorldwideOrganisation ||--|| WorldNewsStory : "has one"
+```
+
+## Proposal
+
+This proposal is that we shall apply the edition workflow to all the remaining non-editionable documents in Whitehall Publisher (and all their associated documents):
```

Comment:
Can I clarify who you mean by "we"? Is it the Publishing Platform team?

Reply:
It wouldn't be the Publishing Platform. We were only really in this space because of the work we were doing around Whitehall rendering. The use of "we" here is more of a royal "we" of GOV.UK, so realistically whichever team is responsible for Whitehall (sorry)!

As much as anything we wanted to write this to advertise that there's a bunch of problems that we've solved and that other content types should be way more straightforward than worldwide orgs were.